docker-compose.windows.yml

```diff
@@ -0,0 +1,30 @@
+version: '3'
+
+services:
+
+  nginx-ui:
+    container_name: nginx-ui
+    build: .
+    image: bifeldy/nginx-ui:latest
+    # ports:
+    #   - 8080:8080
+    volumes:
+      - D:/_data/_docker/nginx:/etc/nginx
+    networks:
+      - bifeldy-net
+
+  nginx:
+    container_name: nginx
+    image: nginx:latest
+    ports:
+      - 8888:80
+    volumes:
+      - D:/_data/_docker/nginx:/etc/nginx
+    networks:
+      - bifeldy-net
+
+networks:
+
+  bifeldy-net:
+    name: bifeldy-net
+    external: true
```

Comment:
Hey @bifeldy, could you please remove this one? It's not generic since it's assumes that
1. a drive D:/ is always the target to mount
2. you are using your username as a network name

btw. everything that's in the repo need maintenance and I don't want to support on issues with docker-compose and windows.

I'm okay with adding it in a seperate section in the README.md to provide an example for people that are using windows.

Best David